.github/workflows/trivy.yml

```diff
@@ -0,0 +1,13 @@
+name: Trivy scan
+on: [pull_request]
+
+jobs:
+  appsec-trivy:
+    name: DSP AppSec Trivy check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - run: touch thurloe.jar
+
+      - uses: broadinstitute/dsp-appsec-trivy-action@v1
```

Comment:
Recommendation: add code comments here that explain why `touch thurloe.jar` is necessary: the `dsp-appsec-trivy-action` builds the docker image, which requires a thurloe.jar to exist.  Maybe add a link to the  `dsp-appsec-trivy-action` repo in case anyone wants to read the source code.